src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/integration/RecallsIntegrationTest.kt

```diff
@@ -0,0 +1,35 @@
+package uk.gov.justice.digital.hmpps.managerecallsapi.integration
+
+import com.ninjasquad.springmockk.MockkBean
+import io.mockk.every
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.managerecallsapi.db.Recall
+import uk.gov.justice.digital.hmpps.managerecallsapi.db.RecallRepository
+import java.util.UUID
+
+class RecallsIntegrationTest : IntegrationTestBase() {
```

Comment:
Should this be merged with the `BookRecallsIntegrationTest`? I assume you were writing this whilst I was writing the other, but nicer to have it clean and tidy.

Reply:
I'm not against it but I can see some arguments for having separate tests suites as well.